vignettes/cookbook_running_the_analysis.Rmd

```diff
@@ -402,16 +417,40 @@ companies_matched %>%
 
 ### Optional: Handling misclassified loans
 
-A loan is considered misclassified, if it is identified as in-scope of a PACTA sector based on the sector classification in the input loan book, but if it turns out that this classification is incorrect based on the counterparty's activities. For example, if a counterparty is classified as a fully in-scope power company, but in reality it is a power distribution company or even active in another sector altogether, such as upstream gas production, the counterparty is misclassified. Misclassification is a problem, because it leads to exaggeratedly low match success rates, that cannot be improved by iterating the matching process. You can search as long as you want, but a company that in reality does not operate any power plants will not be found in the ABCD regardless of how the settings of the matching algorithm are tweaked. The match success rate can only be a useful metric if the denominator, that is the classification in the loan book, is solid. This is also important to understand which loans are truly not matched. This category should only be companies that actually are active in a given in-scope sector, but were not found in the ABCD.
+A misclassified loan occurs when a counterparty is incorrectly identified as belonging to a PACTA sector based on the sector classification in the loan book.
+
+Why does misclassification matter?
+- It artificially lowers match success rates by expecting matches that don’t exist in the ABCD dataset.
+- Iterating the matching process won’t help—if a company doesn’t operate in an in-scope sector, no adjustments will create a valid match.
+- A reliable match success rate requires an accurate denominator, meaning only loans truly within an in-scope sector should be included.
 
-Identifying misclassified loans is usually a manual process, where counterparties that have not been matched are researched to determine if they should be in scope or not. Depending on the size of the loan book, this can be a time-consuming process. It is recommended to start with the unmatched counterparties with the largest loans, as these have the largest impact on the match success rate. You may then have to decide for yourself, if you want to research all unmatched loans or set a cutoff point, e.g. based on the size of the loan.
+Misclassification is typically identified manually by reviewing unmatched counterparties to determine if they should be in scope. Since this process can be time-consuming, it’s best to prioritize unmatched loans with the largest amounts, as they have the greatest impact on the match success rate.
+
+Since misclassified loans cannot be matched, leaving them in the dataset skews results. While correction isn’t mandatory, adjusting large misclassified loans is strongly recommended for more accurate analysis.
```

Comment:
```suggestion
Since misclassified loans cannot be matched, leaving them in the dataset skews results. While correction is not mandatory, adjusting large misclassified loans is strongly recommended for more accurate analysis.
```